ci/README.md

```diff
@@ -26,6 +26,9 @@ Currently there are three build images:
 * `envoyproxy/envoy-build-ubuntu` &mdash; based on Ubuntu 16.04 (Xenial) with GCC 7 and Clang 8 compiler.
 * `envoyproxy/envoy-build-centos` &mdash; based on CentOS 7 with GCC 7 and Clang 8 compiler, this image is experimental and not well tested.
```

Comment:
can you update these line to clang 9? thanks!

Reply:
Addressed in abecfa8.